src/core/gc/config.d

```diff
@@ -15,6 +15,7 @@ __gshared Config config;
 struct Config
 {
     bool disable;            // start disabled
+    bool fork = false;       // optional concurrent behaviour
```

Comment:
I think you should enable it temporarily so the CI tests run with the concurrent GC. It should be restored to `false` before merging, though.